lib/domain/model/extensions/contact/contact_extension.dart

```diff
@@ -8,8 +8,17 @@ extension ContactExtension on Contact {
       displayName: displayName,
       matrixId: matrixId,
       status: status,
-    ),);
+    ),).toSet() ?? {};
 
-    return listContacts?.toSet() ?? {};
+    if (emails == null || emails!.isEmpty) {
+      listContacts.add(PresentationContact(
+        email: null,
```

Comment:
What will happen?

Reply:
no problem, some people maybe dont have email in contact